test/e2e_node/topology_manager_test.go

```diff
@@ -391,6 +391,11 @@ func runTopologyManagerPolicySuiteTests(f *framework.Framework) {
 	runMultipleGuPods(f)
 }
 
+// waitForAllContainerRemoval waits until all the containers on a given pod are really gone.
```

Comment:
Nit on commit message:

```
document why teardowbnSRIOVPod was to wait for all the containers
to be gone before to end, and why is important.
```

should be:
```
document why teardownSRIOVPod has to wait for all the containers
to be gone before to end, and why is important.
```

Reply:
ack, will fix since I want to address the last pending comment

Reply:
actually not sure because I don't want to lose the LGTM if I can help it :\